cni/network/plugin/main.go

```diff
@@ -188,7 +188,17 @@ func main() {
 			reportPluginError(reportManager, tb, err)
 			tb.Close()
 		}
+		return
+	}
 
+	// try and migrate the kvstore
+	if err = netPlugin.MigrateKeyValueStore(); err != nil {
```

Comment:
can you move this call after line 224 .so you dont need to create telemetry buffer and connect .
you can use reportPluginError